source/index.js

```diff
@@ -15,9 +14,6 @@ const levelMapping = [
 	'ansi16m'
 ];
 
-// `color-convert` models to exclude from the Chalk API due to conflicts and such
-const skipModels = new Set(['gray']);
```

Comment:
Why was this removed?

Reply:
Probably because it’s no longer being used.

Reply:
Yeah, no need for blacklist if we're using whitelist.